go-to/code.js

```diff
@@ -0,0 +1,79 @@
+const types = [
```

Comment:
It looks like in other samples we don't check in the `code.js` file. Looks like there are `.gitignore` files in each sample directory: https://github.com/figma/plugin-samples/blob/master/stats/.gitignore